R/shinywrappers.R

```diff
@@ -140,20 +141,20 @@ renderPlot <- function(expr, width='auto', height='auto', res=72, ...,
       }
 
       coordmap <<- list(
-        usr = c(
+        usr = list(
           left = usrCoords[1],
           right = usrCoords[2],
           bottom = usrCoords[3],
           top = usrCoords[4]
         ),
         # The bounds of the plot area, in DOM pixels
-        bounds = c(
-          left = grconvertX(usrBounds[1], 'user', 'nfc') * width,
-          right = grconvertX(usrBounds[2], 'user', 'nfc') * width,
-          bottom = (1-grconvertY(usrBounds[3], 'user', 'nfc')) * height,
-          top = (1-grconvertY(usrBounds[4], 'user', 'nfc')) * height
+        bounds = list(
+          left = grconvertX(usrBounds[1], 'user', 'nfc') * width - 1,
```

Comment:
I think so - I believe R gives pixel coords starting from 1, while in JS the positions start from 0. It is hard to be really precise testing this, but it seems more accurate with the `- 1`.